README.md

```diff
@@ -28,6 +28,6 @@ This repository is used for managing all our dev session resources and [rules](r
 | [Getting Started with Open Source Contribution](sessions/opensource-contribution/) | [Slides](https://slides.com/narainsagar/get-started-opensource-contribution), [Video](https://www.youtube.com/watch?v=xl8YOLCFKGU) | [Narain Sagar](https://github.com/narainsagar) | 12th July, 2017
 | [Jest Basics](sessions/jest/readme.md) | [Slides](https://slides.com/abdulmoiz/deck) | [Muhammad AbdulMoiz](https://github.com/abdulmoizeng) | 5th July, 2017 |
 | [What's new in Angular4](sessions/ng4/) | [Video](https://youtu.be/cKz6O6r9sCY), [Slides](https://docs.google.com/presentation/d/1j_-hN12xylHgbajVlik4GlEptAYNnkKNFGeBqykMPW4/edit#slide=id.p), [Live Demo](https://narainsagar.github.io/ng4-demo), [Code](https://github.com/narainsagar/ng4-demo) | [Narain Sagar](https://github.com/narainsagar) | 29th March, 2017
-| [Debugging Angular2+ Apps](sessions/ng-debugging/) | [Slides](https://slides.com/narainsagar/angular2-debugging) | [Narain Sagar](https://github.com/narainsagar) | 27th January, 2017
+| [Debugging Angular2+ Apps](sessions/ng-debugging/) | [Slides](https://slides.com/narainsagar/angular2-debugging), [Video](#) | [Narain Sagar](https://github.com/narainsagar) | 27th January, 2017
```

Comment:
@mashhoodr this talk video is in office HDD, can we please upload this to our youtube channel? thanks

Reply:
@sohaibnehal

Reply:
Ok, As I'm not our slack channel anymore so can you leave a message to him, I'll also try to email / message on WhatsApp. Thanks.